kenichi127/PoE
# Send mail to reset password

Description / Purpose (checkpoints)
If user forget password, user can send mail to reset password. 
Fixes issue: 
https://docs.google.com/spreadsheets/d/1BHy8K_EvpG4fBLjujxShiZWe2-tNs0Jj/edit#gid=905621989
Task Number :5

How To Test This PR
 1. Go Login Page
 2. Click "Forgot your password?"
 3. input your mail address.
4.click "Send Password Reset Link"
5.Please check your mail box.
 
 **Commands**
Nothing 

 **Screenshots**
 Input mail address
![image](https://user-images.githubusercontent.com/49220772/58681213-14e2f380-839e-11e9-9f00-b02e9f3488d1.png)

mail
![image](https://user-images.githubusercontent.com/49220772/58681241-347a1c00-839e-11e9-9bbf-7a9c72eae645.png)

notes
Please edit your env file like below.

MAIL_DRIVER=smtp
MAIL_HOST=smtp.gmail.com
MAIL_PORT=587
MAIL_USERNAME="Your Gmail"
MAIL_PASSWORD="Your password"
MAIL_ENCRYPTION=tls
MAIL_FROM_ADDRESS="Your gmail"
MAIL_FROM_NAME=PoE

Please go to your google account and security.
and turn on Less secure app access. 

After click "Reset Password".
Please change URL "localhost" to "192.168.99.100:8085".
